docs/api/client_libraries.md

```diff
@@ -8,21 +8,37 @@ nav_order: 1
 
 For anyone looking for COVIDCast data, please visit our [COVIDCast Libraries](covidcast_clients.md).
 
-We are currently working on fully-featured Epidata clients for R and Python. They are not ready
-for release yet, but you can track our development progress and help us test them out at:
+A full-featured Epidata client for R is available at
+[epidatr](https://github.com/cmu-delphi/epidatr) and
+[also on CRAN](https://cran.r-project.org/web/packages/epidatr/index.html).
 
-* [epidatr](https://github.com/cmu-delphi/epidatr)
-* [epidatpy](https://github.com/cmu-delphi/epidatpy)
+We are currently working on a new full-featured Epidata client for Python. It is not ready
+for release yet, but you can track our development progress and help us test it out at
+[epidatpy](https://github.com/cmu-delphi/epidatpy).
 
 In the meantime, minimalist Epidata clients remain available for
 [JavaScript](https://github.com/cmu-delphi/delphi-epidata/blob/master/src/client/delphi_epidata.js),
 [Python](https://github.com/cmu-delphi/delphi-epidata/blob/master/src/client/delphi_epidata.py),
 and
 [R](https://github.com/cmu-delphi/delphi-epidata/blob/master/src/client/delphi_epidata.R).
 The following samples show how to import the library and fetch Delphi's COVID-19
-Surveillance Streams from Facebook Survey CLI for county 06001 and days
+Surveillance Streams from Facebook Survey CLI for county 06001, and days
 `20200401` and `20200405-20200414` (11 days total).
 
+### R
+
+````R
+# [Optional] configure your API key, if desired
+# Interactive. See https://cmu-delphi.github.io/epidatr/articles/epidatr.html#api-keys for details.
+#save_api_key()
+# Import
+library(epidatr)
+# Fetch data
+res <- pub_covidcast('fb-survey', 'smoothed_cli', 'county', 'day', geo_values = '06001',
+                     time_values = c(20200401, 20200405:20200414))
+cat(res)
+````
+
 ### JavaScript (in a web browser)
```

Comment:
while youre here, could you demote the javascript block to right above or below the legacy R item?  (partially because nobody seems to use it, and partially because it doesnt support API keys)

maybe also reorder them to match in the "In the meantime, minimalist" section?  you might also wanna label the R client with "legacy" there too